api/envoy/api/v2/listener/listener.proto

```diff
@@ -87,6 +101,16 @@ message FilterChainMatch {
   // listener in determining a filter chain match.
   // [#not-implemented-hide:]
   google.protobuf.UInt32Value destination_port = 8;
+
+  // If non-empty, a transport protocol to consider when determining a filter chain match.
+  // This value will be compared against transport protocol of a new connection, when it's
+  // detected by one of the listener filters (this needs to be configured separately).
+  //
+  // Valid values include:
+  // * "raw_buffer" - default, used when no transport protocol is detected,
+  // * "tls" - set by :ref:`envoy.listener.tls_inspector <config_listener_filters_tls_inspector>`
+  // when TLS protocol is detected.
```

Comment:
Please check rendered docs. I think this might not render correctly as a bullet list.

Reply:
Fixed.